templates/user/auth/signup_openid_navbar.tmpl

```diff
@@ -0,0 +1,14 @@
+<div class="four wide column">
```

Comment:
To be honest I don't like the horizontal navbar, anyway I guess it makes sense to comply now that it's done.
Could you help with that ? You could file a PR against my branch ..